receiver/prometheusreceiver/metrics_receiver_test.go

```diff
@@ -589,15 +597,49 @@ rpc_duration_seconds_sum{foo="no_quantile"} 101
 rpc_duration_seconds_count{foo="no_quantile"} 55
 `
 
+var target3Page3 = `
```

Comment:
What is this testing?  Please add some comments explaining its utility.

Reply:
@Aneurysm9 thanks. This is added as I changed the scenario of target1 to reflect _high value-failedscrape-lowervalue_. The scenario I removed from target1, I added in target3 using target3page3. 
I have provided the comments there now.

Reply:
I would do that the other way around, then.  Don't change target1 and add the new scenario to target3.  That way only one scenario changes and it is less likely to cause confusion.

Reply:
@Aneurysm9 thanks. I have now made changes accordingly.